source/features/repo-wide-file-finder.tsx

```diff
@@ -11,7 +11,7 @@ async function init(): Promise<void> {
 		<a
 			hidden
 			data-hotkey="t"
-			data-pjax="true"
+			data-pjax="#js-repo-pjax-container"
```

Comment:
Setting the `data-pjax` attribute to `true` caused the hotkey to trigger a "real" navigation instead of an AJAX one, so the handlers added by `file-finder-buffer` weren't called.

This is probably because the element is inserted outside the main "container" (as opposed of the native hotkey which is located on the "Go to file" button).

Fixed it by changing the value of the attribute, mimicking the native link in the repo name heading:
```html
<a data-pjax="#js-repo-pjax-container" href="/sindresorhus/refined-github">refined-github</a>
```

But this could also probably be fixed by inserting the link element directly inside the container.